src/libs/antares/study/parts/thermal/cluster.cpp

```diff
@@ -137,7 +137,11 @@ Data::ThermalCluster::ThermalCluster(Area* parent) :
     forcedLaw(thermalLawUniform),
     plannedLaw(thermalLawUniform),
     PthetaInf(HOURS_PER_YEAR, 0),
-    costsTimeSeries(1, CostsTimeSeries())
+    costsTimeSeries(1, CostsTimeSeries()),
+    maxUpwardPowerRampingRate(0.),
+    maxDownwardPowerRampingRate(0.),
+    powerIncreaseCost(0.),
+    powerDecreaseCost(0.)
```

Comment:
Create sub-class ?
```cpp
struct Ramping {
double maxUpwardPowerRampingRate = 0;
double maxDownwardPowerRampingRate = 0;
// ...
};
Ramping ramping;
```

Reply:
The sub-class is now implemented in commit [b9801fe](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1817/commits/b9801feaf09d246c7f326859a24d4c2b97674d71)